lib/castle_devise/patches/registrations_controller.rb

```diff
@@ -8,8 +8,50 @@ module Patches
     module RegistrationsController
       extend ActiveSupport::Concern
 
-      included do
-        before_action :castle_filter, only: :create
+      # @param klass [self]
+      def self.prepended(klass)
+        klass.class_eval do
+          before_action :castle_filter, only: :create
+        end
+      end
+
+      # PUT /resource
+      def update
+        context = CastleDevise::Context.from_rack_env(request.env, scope_name, resource)
+
+        if resource_class.castle_hooks[:profile_update]
+          begin
+            # TODO: Implement a verification mechanism for this action.
+            CastleDevise.sdk_facade.risk(
+              event: "$profile_update",
```

Comment:
This won't work yet. We have to pass `status: "$attempted"`, add a spec for this and fix our endpoint to also accept `status: $attempted`.

Reply:
Yes, of course. There is another PR prepared for fixing this.